src/components/LoadingAccounts.tsx

```diff
@@ -0,0 +1,17 @@
+import * as rb from 'react-bootstrap'
+
+interface LoadingAccountsProps {
+  amount?: number
+}
+
+export function LoadingAccounts({ amount = 1 }: LoadingAccountsProps) {
+  return (
+    <div>
+      {[...new Array(amount)].map((a, index) => (
+        <rb.Placeholder key={index} as="div" animation="glow" style={{ marginBottom: 6 }}>
```

Comment:
```suggestion
        <rb.Placeholder key={index} as="div" animation="glow">
```

Let's use the CSS from the CSS module here.